src/test/java/jenkins/branch/harness/BranchProjectFactoryImpl.java

```diff
@@ -42,7 +42,7 @@ public BranchProjectFactoryImpl() {}
 
     @Override
     public FreeStyleProject newInstance(Branch branch) {
-        FreeStyleProject job = new FreeStyleProject(getOwner(), branch.getName());
+        FreeStyleProject job = new FreeStyleProject(getOwner(), branch.getEncodedName());
```

Comment:
We [already are](https://github.com/jenkinsci/branch-api-plugin/blob/78766a821ebd05eb62c8eadc7d5d7bd135fa1bcd/src/main/java/jenkins/branch/MultiBranchProject.java#L348-L354). This patch just affects the workspace determination.